Basic-Car-Maintenance/Shared/Settings/Views/AddVehicleView.swift

```diff
@@ -44,6 +44,8 @@ struct AddVehicleView: View {
                     Button {
                         let vehicle = Vehicle(name: name, make: make, model: model)
                         addTapped(vehicle)
+                        // Log vehicle created event to Firebase Analytics
+                        AnalyticsManager.shared.logEventVehicleCreated(.vehicleCreated(RMCVehicleCreatedEvent(vehicleAction: "created", origin: "AddVehicleView", timestamp: Date())))
```

Comment:
You could make the actions enums, so someone doesn't accidentally spell anything wrong

This goes for the actions, and the views ("origin")

Reply:
Done! 

![image](https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/24755875/8c57a70d-1844-43da-8914-a451bc78d804)